grakn-core/src/main/java/ai/grakn/graph/admin/GraknAdmin.java

```diff
@@ -126,6 +126,15 @@
     //------------------------------------- Admin Specific Operations ----------------------------------
 
     /**
+     * Converts a Type Label into a type Id for this specific graph. Mapping labels to ids will differ between graphs
+     * so be sure to use the correct graph when performing the mapping.
+     *
+     * @param label The label to be converted to the id
+     * @return The matching type id or -1 if no such type exists
```

Comment:
`null` breaks your traversals in graql and an `Optional` which is what it used to be led to inserting magic numbers later in your traversals anyway. Not a fan of magic number but since this is internal I think it's okay.